test/oetf/local_images.py

```
@@ -217,13 +225,239 @@ def _load_one(image: ImageSpec, tarball: str) -> None:
+        list(pool.map(_push_one, images, tarball_paths))
+
+
+def ensure_local_registry() -> None:
+    """Start the ``kind-registry`` container if it isn't already running.
+
+    Idempotent: a no-op when the container already exists. Listens on
+    ``127.0.0.1:5001`` so the host can `docker push` to it; KIND nodes
+    reach it as ``http://kind-registry:5000`` after the kind docker
+    network is connected (see :func:`connect_registry_to_kind`).
+    """
+    inspect = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)